set.mm

```diff
@@ -546008,17 +546008,37 @@ extends this idea to multiple additions (TODO).
   $( Alternative definition of ~ whad based on ~ hadifp .  See ~ df-had to
      learn how it is currently introduced.
 
-     Check whether an odd number of inputs are true.  The oddness of two is
-     expressed by the ` \/_ ` operation, and for evenness we can use ` <-> ` .
+     Check whether an odd number of three inputs are true.  The oddness of two
```

Comment:
"The oddness of two" sounds odd - what does it mean?

Reply:
done